### PR TITLE
feat: [ADN-48] Create a token transfer address input component

### PR DESCRIPTION
**Pull request by jinoosss**
Create a token transfer address input component.

### CODE REVIEW

File: packages/adena-extension/src/components/transfer/address-input/address-input.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import AddressInput, { type AddressInputProps } from './address-input';
+import { Meta, StoryObj } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'components/transfer/AddressInput',
+  component: AddressInput,
+} as Meta<typeof AddressInput>;
+
+const addressBookInfos = [
```
Comment (luvhaeun) [inline packages/adena-extension/src/components/transfer/address-input/address-input.stories.tsx:10]: Is this actually used in the storybook?

File: packages/adena-extension/src/components/transfer/address-input/address-input.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    addressBookId: string;
+    name: string;
+    description: string;
+  }[];
+  onClickInputIcon: (selected: boolean) => void;
+  onChangeAddress: (address: string) => void;
+  onClickAddressBook: (addressBookId: string) => void;
+}
+
+const AddressInput: React.FC<AddressInputProps> = ({
```
Comment (luvhaeun) [inline packages/adena-extension/src/components/transfer/address-input/address-input.tsx:25]: Screenshot 2023-04-21 at 7 32 59 PM
I think the borders are weird

**Review by luvhaeun — COMMENTED** [review]
Complete the first review @jinoosss

File: packages/adena-extension/src/components/transfer/address-input/address-input.stories.tsx (hunk 1 above)
Comment (jinoosss) [reply packages/adena-extension/src/components/transfer/address-input/address-input.stories.tsx:10]: This variable is the address book information when the 'opened' option is enabled.

**Review by luvhaeun — APPROVED** [review]

**Pull request merged**